### ISSUE TITLE
Component fugitives model not working for field 85

### ISSUE THREAD

**Issue by arbrandt** [arbrandt/OPGEE#272]
Field 85 breaks the crude oil storage tank fugitive emissions estimate in fractional loss rate when on component fugitives model

**Comment by arbrandt** [comment]
I turned off the component level model to work on the diluent issues. Component model also seems to drive emissions quite a bit higher than the site model for field 0.

**Comment by JSRuthe** [comment]
The issue here is that the tank flashing emission loss rate is calculated as (random draw from HARC dataset)/(gas at wellhead)

The problem is that the wellhead doesn't exist in oil sands production. In this case its a bitumen mine.

@arbrandt  any suggestions for maybe an "if statement" for oil sands processes? The problem is that there really isn't any gas production at oil sands mines to use in the denominator

**Comment by qlangfitt** [comment]
Just noticed that #274, which I reported this morning, is about the same issue. I will close that issue since it's a duplicate.

In order to continue running fields I patched this by adjusting the fugitive emissions in the outflow table on the Crude Oil Storage sheet. This leaves the fractional loss with a DIV0 error, but allows the model to still run fine and calculate results. If you didn't want that unsightly error, could use an IF statement on the VF - component sheet to simply say "N/A" or something like that for oil sands fields, given that the fix below is implemented.

Here's what I put in Crude Oil Storage Cell Q20:
=IF(SUM(Oil_sands_mine_int_01,Oil_sands_mine_nonint_01)=0,IF($N$1="ON",M50*'Flow Sheet'!AF16,0),((1-f_FG_CS_VRU/100)*'VF - Component'!H307*Oil_prod)/1000)
